elasticapm/instrumentation/packages/asyncio/aioredis.py

```diff
@@ -73,7 +74,8 @@ def call(self, module, method, wrapped, instance, args, kwargs):
 class RedisConnectionInstrumentation(AbstractInstrumentedModule):
     name = "aioredis"
 
-    instrument_list = (("aioredis.connection", "RedisConnection.execute"),)
+    instrument_list = (("aioredis.connection", "RedisConnection.execute"),
+                       ("aioredis.pool", "ConnectionsPool.execute_pubsub"))
 
     def call(self, module, method, wrapped, instance, args, kwargs):
         span = execution_context.get_span()
```

Comment:
this is missing ``` async_capture_span``` call hence it would not create a span in APM. Also, I think this should also be async and return needs to have await with it @beniwohli @Sparkycz